install/empty_data.php

```diff
@@ -0,0 +1,8198 @@
+      'mode'          => 1,
+      'lastrun'       => null,
+      'logs_lifetime' => 10,
+   ], [
+      'id'            => 29,
+      'itemtype'      => 'SavedSearch_Alert',
+      'name'          => 'savedsearchesalerts',
+      'frequency'     => 86400,
+      'param'         => null,
+      'state'         => 0,
+      'mode'          => 1,
+      'lastrun'       => null,
+      'logs_lifetime' => 10,
+   ], [
+      'id'            => 30,
+      'itemtype'      => 'Telemetry',
+      'name'          => 'telemetry',
+      'frequency'     => 2592000,
+      'param'         => null,
+      'state'         => 0,
+      'mode'          => 1,
+      'lastrun'       => null,
+      'logs_lifetime' => 10,
+   ], [
+      'id'            => 31,
+      'itemtype'      => 'Certificate',
+      'name'          => 'certificate',
+      'frequency'     => 86400,
+      'param'         => null,
+      'state'         => 0,
+      'mode'          => 1,
+      'lastrun'       => null,
+      'logs_lifetime' => 10,
+   ], [
+      'id'            => 32,
+      'itemtype'      => 'OlaLevel_Ticket',
+      'name'          => 'olaticket',
+      'frequency'     => 300,
+      'param'         => null,
+      'state'         => 1,
+      'mode'          => 1,
+      'lastrun'       => '2014-06-18 08:02:00',
+      'logs_lifetime' => 30,
+   ], [
+      'id'            => 33,
+      'itemtype'      => 'PurgeLogs',
+      'name'          => 'PurgeLogs',
+      'frequency'     => 604800,
+      'param'         => 24,
+      'state'         => 1,
+      'mode'          => 2,
+      'lastrun'       => null,
+      'logs_lifetime' => 30,
+   ], [
+      'id'            => 34,
+      'itemtype'      => 'Ticket',
+      'name'          => 'purgeticket',
+      'frequency'     => 43200,
+      'param'         => null,
+      'state'         => 0,
+      'mode'          => 1,
+      'lastrun'       => null,
+      'logs_lifetime' => 30,
+   ], [
+      'id'            => 35,
+      'itemtype'      => 'Document',
+      'name'          => 'cleanorphans',
+      'frequency'     => 43200,
+      'param'         => null,
+      'state'         => 0,
+      'mode'          => 1,
+      'lastrun'       => null,
+      'logs_lifetime' => 30,
+   ],
+];
+
+$tables['glpi_devicememorytypes'] = [
+   [
+      'id'   => 1,
+      'name' => 'EDO',
+   ],
+   [
+      'id'   => 2,
+      'name' => 'DDR',
+   ],
+   [
+      'id'   => 3,
+      'name' => 'SDRAM',
+   ],
+   [
+      'id'   => 4,
+      'name' => 'SDRAM-2',
+   ],
+];
+
+$tables['glpi_devicesimcardtypes'] = [
+   [
+      'id'   => 1,
+      'name' => 'Full SIM',
+   ],
+   [
+      'id'   => 2,
+      'name' => 'Mini SIM',
+   ],
+   [
+      'id'   => 3,
+      'name' => 'Micro SIM',
+   ],
+   [
+      'id'   => 4,
+      'name' => 'Nano SIM',
+   ],
+];
+
+$tables['glpi_displaypreferences'] = [
+   [
+      'itemtype' => 'Computer',
+      'num'      => '4',
+      'rank'     => '4',
+       //'is_main'  => '1'
```

Comment:
I've kept those comments because they'll be very useful to me on another feature I'm working on. I'll drop them before merging ;)

Reply:
Dropped :)